server/server_test.go

```diff
@@ -410,160 +407,6 @@ func TestCountOpenFiles(t *testing.T) {
 	}
 }
 
```

Comment:
@raskle I'm not opposed to keeping this test (and I'm indifferent on splitting out to an integration tag, although that doesn't really seem necessary). We would also need to keep the `httpbroadcast` package. That's not really a problem either except that that package requires `node.InternalHost` in method `sendNodeMessage()`, but `node.InternalHost` no longer exists. So we would need to make some modifications to `httpbroadcast` so that it's not dependent upon those configuration values that are no longer supported, and so that it's aware of the internalHosts (also no longer part of the configuration).